webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/ComplianceApi.scala

```diff
@@ -336,6 +336,7 @@ class ComplianceAPIService(
                     ByRuleNodeCompliance(
                       nodeId
                       , nodeInfos.get(nodeId).map(_.hostname).getOrElse("Unknown node")
+                      , ComplianceLevel.sum(components.find(_._1 == nodeId).map(_._2.compliance).toList)
```

Comment:
i'm surprised by this line
the .find().map() returns an option. the toList hides it

Reply:
if there is 0 component (none found), it seems correct to return "n/a". 
But I'm unsure why we can't have several component here. 
And it could be done in one pass with `collectFirst { case (id, v) if(id == nodeId) => v.compliance }`

OK, it was a map before being "toSeq"